.github/workflows/update-wiki-docs.yml

```diff
@@ -0,0 +1,56 @@
+name: Update wiki docs
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+concurrency:
+  group: wiki
+  cancel-in-progress: true
+
+permissions:
```

Comment:
I think more permissions need to be included? 
`permissions:
  contents: write
  pull-requests: write
  wiki: write`

Reply:
"wiki" is an unknown permission key. 

https://github.com/marketplace/actions/github-wiki-action